hack/init_build.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -u
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+. "${ROOT}/hack/utilities.sh" || { echo 'Cannot load bash utilities.'; exit 1; }
+
+VERSIONS="/tmp/.versions"
+GO_VERSION='1.7.3'
+HELM_VERSION='v2.0.0'
+GLIDE_VERSION='v0.12.3'
+
+function update-golang() {
+  # Check version of golang
+  local current_go_version="$(cat "${VERSIONS}/go" || echo "unknown")"
+
+  if [[ "${current_go_version}" == "${GO_VERSION}" ]]; then
+    echo "Golang is up-to-date: ${current_go_version}"
+  else
+    echo "Upgrading golang ${current_go_version} to ${GO_VERSION}"
+
+    # Install new golang.
+    local golang_url='https://storage.googleapis.com/golang'
+    rm -rf /usr/local/go \
+      && curl -sSL "${golang_url}/go${GO_VERSION}.linux-amd64.tar.gz" \
+         | tar -C /usr/local -xz \
+      || { echo "Cannot upgrade golang to ${GO_VERSION}"; return 1; }
+  fi
+  return 0
```

Comment:
since I don't think its needed, and to be consistent with the other funcs, let's remove this return.